### PR TITLE
Reproject NDCubeSequence Documentation

### PR DESCRIPTION
**Pull request by adwaitbhope**
### Description

Added docs for reprojecting `NDCubeSequence` (#436).

### CODE REVIEW

File: docs/reproject.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  >>> wcs2.wcs.ctype = 'WAVE', 'HPLT-TAN', 'HPLN-TAN'
+  >>> wcs2.wcs.cunit = 'Angstrom', 'deg', 'deg'
+  >>> wcs2.wcs.cdelt = 0.2, 0.5, 0.4
+  >>> wcs2.wcs.crpix = 0, 2, 2
+  >>> wcs2.wcs.crval = 10, 0.5, 1
+
+  >>> # Instantiate NDCubes with supporting data.
+  >>> cube0 = NDCube(data0, wcs=wcs0)
+  >>> cube1 = NDCube(data1, wcs=wcs1)
+  >>> cube2 = NDCube(data2, wcs=wcs2)
```
Comment (adwaitbhope) [inline docs/reproject.rst:105]: These three WCS objects are still essentially the same coordinates. Should we initialize the cubes with the same object or introduce some changes in the WCS?

I think passing on the same WCS instance will not highlight that `combine_cubes()` reprojects the cubes.

Comment (Cadair) [reply docs/reproject.rst:105]: Yeah I think we should probably change them. Maybe change the crpix values around a bit? (Also the HPLN/LT coordinates should really have CRVAL be 0,0).

Comment (DanRyanIrish) [reply docs/reproject.rst:105]: Any updates on this @adwaitbhope?

Comment (adwaitbhope) [reply docs/reproject.rst:105]: Added a commit now

**Comment by DanRyanIrish** [comment]
I think this looks good to merge.  But the conflicts need to be resolved first.

**Comment by adwaitbhope** [comment]
Yes, shall do that once #436 is merged.

**Comment by DanRyanIrish** [comment]
:+1:

**Pull request closed without merging**